concordium-node/README.md

```diff
@@ -92,38 +97,71 @@ Environment variables only apply to the default build. This links with shared Ha
 
 - The node built with Haskell library auto-discovery is not suitable for distribution to other
   machines. It is a dynamically linked binary with a large number of shared library dependencies.
 
+### MacOS Specific
+
+You may need to add the following entries to your `~/.stack/config.yaml` for the libraries installed via `brew`:
+
+```yaml
+extra-lib-dirs:
+- /opt/homebrew/lib
+
+extra-include-dirs:
+- /opt/homebrew/include/
+```
+
+### M1 MacOS Specific
+
+You may need to add the following entry to your `~/.stack/config.yaml` for the `libffi` include:
+
+```yaml
+extra-include-dirs:
+- /Library/Developer/CommandLineTools/SDKs/MacOSX12.3.sdk/usr/include/ffi/
+```
+
+To determine the exact path to the `libffi` include directory, run the following command:
+
+```sh
+pkg-config --cflags libffi
+```
+
 ## Building on Windows
 
 ### Dependencies
 
 Before building the node, you should install the following dependencies:
 
-* Haskell [stack](https://docs.haskellstack.org/en/stable/install_and_upgrade/)
-* [Rust](https://www.rust-lang.org/tools/install)
-  * For building the node, the toolchain `1.53.0-x86_64-pc-windows-gnu` is required, which can be installed with the command: `rustup toolchain install 1.53.0-x86_64-pc-windows-gnu`.
-  * For building the node runner service (optional), the toolchain `1.53.0-x86_64-pc-windows-msvc`  is required, which can be installed with the command: `rustup toolchain install 1.53.0-x86_64-pc-windows-msvc`.
-* [flatc](https://github.com/google/flatbuffers/releases/tag/v2.0.0) 2.0.0 (should be in the path)
-* [protoc](https://github.com/protocolbuffers/protobuf/releases) >= 3.7.1
-* PostGreSQL and LMDB should be installed under `stack`'s `msys2` installation, which can be done with the following commands:
-```
+- Haskell [stack](https://docs.haskellstack.org/en/stable/install_and_upgrade/)
+- [Rust](https://www.rust-lang.org/tools/install)
+  - For building the node, the toolchain `1.53.0-x86_64-pc-windows-gnu` is required, which can be installed with the command: `rustup toolchain install 1.53.0-x86_64-pc-windows-gnu`.
+  - For building the node runner service (optional), the toolchain `1.53.0-x86_64-pc-windows-msvc`  is required, which can be installed with the command: `rustup toolchain install 1.53.0-x86_64-pc-windows-msvc`.
+- [flatc](https://github.com/google/flatbuffers/releases/tag/v2.0.0) 2.0.0 (should be in the path)
+- [protoc](https://github.com/protocolbuffers/protobuf/releases) >= 3.7.1
+- PostGreSQL and LMDB should be installed under `stack`'s `msys2` installation, which can be done with the following commands:
+
+```bash
 stack exec -- pacman -Syuq --noconfirm
 stack exec -- pacman -Syq mingw-w64-x86_64-postgresql mingw-w64-x86_64-lmdb --noconfirm
 ```
-* If building the installer, the [Wix Toolset](https://wixtoolset.org/releases/) is required, and should be in the path.
+
+- If building the installer, the [Wix Toolset](https://wixtoolset.org/releases/) is required, and should be in the path.
```

Comment:
Also here,.

Reply:
It's a markdown linter fix, the symbol for the list items should be. consistent in a file, either. `-` or `*`